students

```diff
@@ -1,2 +1,3 @@
 Ihor Kandaurov - https://github.com/lbrdev/Lesson1.LS.ld/
 Nastya Bezrukavaya - https://github.com/anAstyQ?tab=repositories
+Yaroslav Pavlinskiy - https://github.com/yarikpavlin/MyRepository

```

Comment:
@yarikpavlin , please review your created repo:
1) insert your name in LICENSE file (at the bottom)
2) git config -- list do not contain your email
3) repo readme should have some description and repo should have some meaningful name